platzky/admin/admin.py

```
@@ -0,0 +1,25 @@
+from os.path import dirname
+from flask import Blueprint, render_template, request, session
```

Ruff (0.8.2)

`flask.request` imported but unused

Remove unused import: `flask.request`

(F401)